setup.py

```diff
@@ -248,7 +248,7 @@ def do_setup():
         scripts=['airflow/bin/airflow'],
         install_requires=[
             'alembic>=0.8.3, <0.9',
-            'bleach==2.1.2',
+            'bleach==2.1.3',
```

Comment:
If we are changing this do we still want it as an exact requirement? Would `bleach=~2.1.3` be a better plan?

Reply:
good point, I have changed it as you proposed.